WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -127,6 +131,9 @@ public void end() {
         properties.put(KEY_BLOG_TYPE, mBlogType);
         properties.put(KEY_SESSION_ID, mSessionId);
         properties.put(KEY_HAS_UNSUPPORTED_BLOCKS, mHasUnsupportedBlocks ? "1" : "0");
+        if (mUnsupportedBlocks != null && mUnsupportedBlocks.size() > 0) {
+            properties.put(KEY_UNSUPPORTED_BLOCKS, mUnsupportedBlocks);
```

Comment:
Question:
Is it OK/Better to not send this key at all if there are no unsupported block VS sending an empty array?

Reply:
> Question:
> Is it OK/Better to not send this key at all if there are no unsupported block VS sending an empty array?

Will let @koke reply on this, since he has more experience querying data.

Reply:
I don't think it's better. I haven't used array properties yet in queries, but my guess is that either it doesn't matter, or it's best to avoid null values. I lean towards the first, since we would have null values anyway from previous versions, but an empty array is more explicit